### PR TITLE
Update bootstrap-table-ko-KR.js

### PR DESCRIPTION
**Pull request by ablyeom**
**🤔Type of Request**
- [ ] **Bug fix**
- [ ] **New feature**
- [x] **Improvement**
- [ ] **Documentation**
- [ ] **Other**

**🔗Resolves an issue?**

**📝Changelog**

- [x] **Core**
- [ ] **Extensions**

**💡Example(s)?**

**☑️Self Check before Merge**

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by UtechtDustin** [comment]
Please fix the linting error (CI) and we will merge this :-)

**Comment by djhvscf** [comment]
This is the error @ablyeom 

bootstrap-table-ko-KR.js
  107:77  error  Newline required at end of file but not found  eol-last

https://app.travis-ci.com/github/wenzhixin/bootstrap-table/jobs/549921057#L316

As @UtechtDustin please fix the error and then we can proceed.

File: src/locale/bootstrap-table-ko-KR.js [hunk 1]
```diff
@@ -60,47 +61,47 @@ $.fn.bootstrapTable.locales['ko-KR'] = $.fn.bootstrapTable.locales['ko'] = {
...
+    return '컨트롤 숨기기'
   },
   formatFilterControlSwitchShow () {
-    return 'Show controls'
+    return '컨트롤 보기'
   }
 }
 
-$.extend($.fn.bootstrapTable.defaults, $.fn.bootstrapTable.locales['ko-KR'])
+$.extend($.fn.bootstrapTable.defaults, $.fn.bootstrapTable.locales['ko-KR'])
```
Comment (wenzhixin) [inline src/locale/bootstrap-table-ko-KR.js:107]: There is no need to modify this line.

Comment (wenzhixin) [reply src/locale/bootstrap-table-ko-KR.js]: Updated

**Review by wenzhixin — APPROVED** [review]

**Pull request merged**